### PR TITLE
TWM-106 Deploying the app

### PR DESCRIPTION
**Pull request by sensei100**
- We need the assets in production, so this adds those steps back to the Dockerfile

### CODE REVIEW

File: .docker/app/Dockerfile [hunk 1]
```diff
@@ -6,8 +6,8 @@ COPY . .
 
 USER root
 
-RUN apk add --update --no-cache bash=5.1.0-r0 tzdata=2021a-r0 libxslt-dev=1.1.34-r0 mariadb-dev=10.5.8-r0 && \
-    apk add --no-cache --virtual build-dependencies git=2.30.0-r0 build-base=0.5-r2 && \
+RUN apk add --no-cache bash=5.0.17-r0 tzdata=2020f-r0 libxslt-dev=1.1.34-r0 mariadb-dev=10.4.15-r0 nodejs=12.20.1-r0 yarn=1.22.4-r0 && \
```
Comment (dkinzer) [inline .docker/app/Dockerfile:9]: @sensei100 can we move nodejs and yarn to the --virtual build-dependencies section and then move line 17 where we delete build-depencies to be right before `rm -f /var/cache/apk/`?

Comment (sensei100) [reply .docker/app/Dockerfile]: @dkinzer Okay, I pushed up that change.

File: .docker/app/Dockerfile [hunk 2]
```diff
@@ -6,18 +6,22 @@ COPY . .
...
-    apk del build-dependencies && \
     chown nobody -R /app/tmp && \
     chown nobody -R /app/log && \
-    rm -rf /var/cache/apk/
+    apk del build-dependencies && \
+    rm -rf /var/cache/apk/ && \
+    yarn install --check-files && \
+    RAILS_ENV=production bundle exec rake assets:precompile && \
+    rm -rf app/node_modules && \
+    rm -rf app/tmp/*
```
Comment (dkinzer) [inline .docker/app/Dockerfile:24]:
```suggestion
    yarn install --check-files && \
    RAILS_ENV=production bundle exec rake assets:precompile && \
    rm -rf app/node_modules && \
    rm -rf app/tmp/*
    apk del build-dependencies && \
    rm -rf /var/cache/apk/ && \
```

Comment (dkinzer) [reply .docker/app/Dockerfile:24]: @sensei100 sorry I meant to move ` apk del build-dependencies && \` down

File: .docker/app/Dockerfile [hunk 3]
```diff
@@ -6,18 +6,22 @@ COPY . .
...
-    apk del build-dependencies && \
     chown nobody -R /app/tmp && \
     chown nobody -R /app/log && \
-    rm -rf /var/cache/apk/
+    yarn install --check-files && \
+    RAILS_ENV=production bundle exec rake assets:precompile && \
+    rm -rf app/node_modules && \
+    rm -rf app/tmp/*
+    apk del build-dependencies && \
+    rm -rf /var/cache/apk/ && \
```
Comment (dkinzer) [inline .docker/app/Dockerfile:24]: argh , sorry...  we need to also update && \ now.
```suggestion
    rm -rf app/tmp/* && \
    apk del build-dependencies && \
    rm -rf /var/cache/apk/
```

**Review by dkinzer — APPROVED** [review]

**Pull request merged**